enqueue/amqp-ext/0.3/etc/packages/enqueue_amqp.yaml

```diff
@@ -0,0 +1,23 @@
+enqueue:
+    transport:
+        default: 'amqp'
+        amqp:
+            host: '%env(ENQUEUE_AMQP_HOST)%'
+            port: '%env(ENQUEUE_AMQP_PORT)%'
+            login: '%env(ENQUEUE_AMQP_LOGIN)%'
+            password: '%env(ENQUEUE_AMQP_PASSWORD)%'
+            vhost: '%env(ENQUEUE_AMQP_VHOST)%'
+
+#        If you use RabbitMQ broker you can replace amqp transport with this one. It enables specific features like priorities, delays.
+#        default: 'rabbitmq_amqp'
+#        rabbitmq_amqp:
+#
+#            host: '%env(ENQUEUE_AMQP_HOST)%'
+#            port: '%env(ENQUEUE_AMQP_PORT)%'
+#            login: '%env(ENQUEUE_AMQP_LOGIN)%'
+#            password: '%env(ENQUEUE_AMQP_PASSWORD)%'
+#            vhost: '%env(ENQUEUE_AMQP_VHOST)%'
```

Comment:
The whole config seems to be the same as the default one, except for the option. What about adding the comment on the default config with the explanation you have above?